scripts/pre-commit.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
```

Comment:
`exec task lint` should be the shortest option here? maybe it needs bash for `exec`;

Reply:
unused with lefthook